app/models/manageiq/providers/openstack/inventory/parser/cloud_manager.rb

```diff
@@ -220,7 +220,7 @@ def orchestration_template(stack)
     template = collector.orchestration_template(stack)
     if template
       o = persister.orchestration_templates.find_or_build(stack.id)
-      o.type = stack.template.format == "HOT" ? "OrchestrationTemplateHot" : "OrchestrationTemplateCfn"
```

Comment:
@bzwei why did we have CFN template format here, when people are not using HOT?

Reply:
Yes, openstack can take CFN template too. With this refactored class, now Openstack's template support both JSON (CFN) and YAML (HOT) format.